src/Features/CSharp/Portable/InvertIf/CSharpInvertIfCodeRefactoringProvider.cs

```diff
@@ -201,6 +204,10 @@ protected override StatementSyntax AsEmbeddedStatement(IEnumerable<StatementSynt
 
                 updatedIf = updatedIf.WithElse(elseClause);
             }
+            else if (!isSingleLine && shouldAddElasticTrivia)
```

Comment:
nit: rename to shouldAddElasticNewLine.  There is the concept of non-newline-elastictrivia, so this parameter is a bit unclear.